lnwallet/channel.go

```diff
@@ -1573,6 +1573,10 @@ func (lc *LightningChannel) remoteLogUpdateToPayDesc(logUpdate *channeldb.LogUpd
 	case *lnwire.UpdateFulfillHTLC:
 		ogHTLC := localUpdateLog.lookupHtlc(wireMsg.ID)
 
+		if ogHTLC.addCommitHeightLocal == 0 {
+			ogHTLC.addCommitHeightLocal = commitHeight - 1
```

Comment:
why do we use `-1` here?

Reply:
I think `commitHeight` would probably work, should try that instead. I just made it one less than the remove height, since it logically *must* be at least one less.

Reply:
These add heights are only used to track when things have been locked into both commitments. So if we know it's already on our commitment, then any thing at our height or below is fine. However, bugs would be introduced if we used a height _higher_ than the proper height, since that could cause us to attempt to forward the HTLCs, or try to settle/fail them before they've been fully locked into both commitments.

Reply:
Will change to set to just `commitHeight` then?

Reply:
Either should work I think, but if this is what we've been testing against so far, then we may just want to leave it.

Reply:
Keeping as is.